### DISCUSSION TITLE
the scroll_to is not yet implemented

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#1905] [General]
The scroll_to is not yet implemented, is there any way to scroll the row or column to create an auto scrolling animation using which I can make the rows and columns scroll continuously.

**Comment by FeodorFitsner** [comment]
See https://flet.dev/docs/controls/row#auto_scroll and https://flet.dev/docs/controls/row#scroll_tooffset-delta-key-duration-curve.
